selectrum.el

```diff
@@ -492,6 +493,16 @@ provided, rather than providing one of their own."
     (setq selectrum--current-candidate-index
           (1- (length selectrum--refined-candidates)))))
 
+(defun selectrum-kill-ring-save ()
+  "Save current candidate to kill ring.
+Or if there is an active region, save the region to kill ring."
+  (interactive)
+  (if (region-active-p)
```

Comment:
The docstring recommends to use `use-region-p` instead, which I think is reasonable. Also, can we delegate to `kill-ring-save` directly if `transient-mark-mode` is disabled, so that we can avoid breaking the normal kill-yank functionality in this case?

Reply:
Done.